### PR TITLE
Changed politeness histogram plots and the way we present n-grams

### PR DESCRIPTION
**Pull request by sophieball**

### CODE REVIEW

File: plot_politeness.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+
+scores = pd.read_csv("~/Desktop/polite_strategies_label_1 (1).csv", index_col=0)
+#scores = pd.read_csv("G_polite_1.csv")
+print(scores)
+scores = scores.drop(index=["HASHEDGE"], axis=0)
+scores = scores.to_dict("dict")["Averages"]
+#scores = scores.rename(columns={"index": "type", "Average": "Average"})
+#get_polite = lambda a:type_colors[a]
+#scores["politeness"] = scores["type"].map(get_polite)
```
Comment (CaptainEmerson) [inline plot_politeness.py:120]: rm here, above?

File: src/plot_politeness.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
+# Lint as: python3
```
Comment (CaptainEmerson) [inline src/plot_politeness.py:1]: Wait, this file appears twice in this PR?

File: plot_politeness.py (hunk 1 above)
Comment (sophieball) [reply plot_politeness.py]: Removed! I removed the whole file actually.

File: src/plot_politeness.py (hunk 2 above)
Comment (sophieball) [reply src/plot_politeness.py:1]: :/ the other one is the duplicate

**Comment by CaptainEmerson** [comment]
The z-scores don't get sorted correctly in my ngram column:
```
abs(z-score)
1.127727465
1.127727465
1.127727465
0.9593232635
0.9267823327
0.8228805307
0.7318181857
0.6578762715
0.6578762715
0.357885226
3.196124698
2.247262102
2.211874137
2.055848259
1.803939237
1.803939237
1.803275636
1.774687938
1.694652204
1.54177375
```

Or maybe the formatting is wrong? Maybe some exponents are getting dropped?

**Comment by sophieball** [comment]
> The magic numbers make it seem like it would break easily if the input changes

It comes from here: http://languagelog.ldc.upenn.edu/myl/Monroe.pdf (Figure 2)

@bvasiles

**Comment by CaptainEmerson** [comment]
I can approve this once the sorting issue is resolved.

**Comment by sophieball** [comment]
image


I've been trying macroaveraging. Counts of politeness strategies divided by # sentences.. so now every strategy is divided by # sentences

**Comment by CaptainEmerson** [comment]
I've added the results for G data in the Google drive. The unigrams are a bit weirder than they were before, including "and" as the top indicator of pushback.

**Pull request merged**